src/LowercaseSentenceTokenizer.java

```diff
@@ -29,8 +30,36 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    * @return a list of tokens, where each token is a word or a period
    */
   public List<String> tokenize(Scanner scanner) {
-    // TODO: Implement this function to convert the scanner's input to a list of words and periods
-    return null;
+    List<String> list = new ArrayList<String>();
+    
+    // Loops through the provided text, ending when the text no longer has another word in it
+    while(scanner.hasNext()){
+      // Grabs the next word in the text
+      String token = scanner.next().toLowerCase();
+
+      // Checks to see if the word has a "." in it
+      if(token.contains(".")){
```

Comment:
I like that you're adding comments, but you can consider leaving out ones that are a bit redundant here and throughout your program. It's already clear what the line of code is doing, the comment doesn't add any extra explanation